### PR TITLE
upgrade Snyk CLI version to v1.801.0

### PR DESCRIPTION
**Pull request by glours**
Signed-off-by: Guillaume Lours <guillaume.lours@docker.com>
**- What I did**
Upgrade Snyk CLI to version `v1.801.0` and pass `--nested-jars-path=2` by default when scanning an image

**- A picture of a cute animal (not mandatory)**
image

### CODE REVIEW

File: internal/provider/provider.go [hunk 1]
```diff
@@ -155,6 +155,8 @@ func WithGroupIssues() Ops {
 func WithAppVulns() Ops {
 	return func(provider *Options) error {
 		provider.flags = append(provider.flags, "--app-vulns")
+		// We started with a default depth value of 2
```
Comment (thaJeztah) [inline internal/provider/provider.go:158]: This limits the "maximum depth" to search? Would setting higher values cause a large overhead? (wondering if there would be downsides to setting it higher).

**Review by thaJeztah — APPROVED** [review]
SGTM, but left a comment/question

**Review by StefanScherer — APPROVED** [review]
LGTM

**Review by chris-crone — APPROVED** [review]
LGTM

File: internal/provider/provider.go (hunk 1 above)
Comment (glours) [reply internal/provider/provider.go:158]: See the internal Slack thread which explains this choice

Comment (StefanScherer) [reply internal/provider/provider.go:158]: we don't know exactly, but we know that it works with the default value which is 1, so we thought 2 is better :-)

Comment (thaJeztah) [reply internal/provider/provider.go:158]: As long as `2` doesn't mean "skip those that are only nested `1` level deep" 😬 😅

**Pull request merged**